src/coord/src/coord.rs

```diff
@@ -2056,10 +2083,7 @@ where
         if self.symbiosis.is_some() {
             // In symbiosis mode, we enforce serializability by forcing all
             // PEEKs to peek at the latest input time.
-            // TODO(benesch): should this be saturating subtraction, and what should happen
-            // when `self.last_assigned` is zero?
-            assert!(self.last_assigned > 0);
-            return Ok(self.last_assigned - 1);
```

Comment:
I don't have a good sense myself if this breaks anything!

Reply:
The only thing that uses symbiosis is SLT, so should be pretty obvious if something is broken!